modules/hdbt_content/hdbt_content.tokens.inc

```diff
@@ -60,7 +60,14 @@ function hdbt_content_tokens(
     if ($name === 'picture' && $type === 'tpr_unit' && !empty($data[$type])) {
       /** @var \Drupal\helfi_tpr\Entity\Unit $entity */
       $entity = $data[$type];
-      $replacements[$original] = $entity->getPictureUrl() ?? $default_image;
+      $image = NULL;
+
+      try {
+        $image = $entity->getPictureUrl();
+      }
+      catch (\Exception) {
```

Comment:
Is it necessary to catch an exception here? Could we just check whether `$entity->getPictureUrl()` returns `FALSE`?

Reply:
No, but this exception check could be moved to TPR module instead: https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/blob/main/src/Entity/Unit.php#L167